features/new.feature

```diff
@@ -109,8 +109,10 @@ Feature: create a template for a new zendesk app
  },
  "defaultLocale": "en",
  "private": true,
- "location": { "zendesk":
-   { "ticket_sidebar": "assets/iframe.html" }
+ "location": {
+   "support": {
+    "ticket_sidebar": "assets/iframe.html"
```

Comment:
done, feel free to implement a linter and add it to the travis run if this kinda thing bothers you